lib/configure.js

```diff
@@ -319,6 +348,9 @@ function configure (gyp, argv, callback) {
       argv.push('-Dlibrary=shared_library')
       argv.push('-Dvisibility=default')
       argv.push('-Dnode_root_dir=' + nodeDir)
+      if (process.platform === 'aix') {
+        argv.push('-Dnode_exp_file=' + node_exp_file)
```

Comment:
I'd prefer that we still add the flag so that we can see what occurred.  In verbose mode we'll be able to see what the value for -Dnode_exp_file is